src/plugins/data/common/es_query/kuery/node_types/node_builder.ts

```diff
@@ -16,12 +16,10 @@ export const nodeBuilder = {
       nodeTypes.literal.buildNode(false),
     ]);
   },
-  or: ([first, ...args]: KueryNode[]): KueryNode => {
```

Comment:
changes the "and" and "or" functions to produce their output linearly, instead of recursively.  Note, the KQL PEG parser still produces equivalent expressions recursively.  Presumably these functions previously worked recursively to match the KQL parser.